src/contrib/cluster/Akka.Cluster.Tools/Singleton/ClusterSingletonManager.cs

```diff
@@ -619,6 +575,17 @@ public ClusterSingletonManager(Props singletonProps, object terminationMessage,
                     .GetLease(_singletonLeaseName, settings.LeaseSettings.LeaseImplementation, _cluster.SelfAddress.HostPort());
                 _leaseRetryInterval = settings.LeaseSettings.LeaseRetryInterval;
             }
+            
+            // Added in v1.5.27 to signal to users who were considering AppVersion
+            // in their singleton placement decisions that we don't do that any more
+#pragma warning disable CS0618 // Type or member is obsolete
+            if (settings.ConsiderAppVersion)
+#pragma warning restore CS0618 // Type or member is obsolete
+            {
+                Log.Warning("As of Akka.NET v1.5.27, The 'ConsiderAppVersion' setting is no longer supported and will " +
```

Comment:
Add a warning for users who had `ConsiderAppVersion` enabled - it was disabled by default, so it shouldn't affect too many people.